src/module/Job/hooks/useJob.ts

```diff
@@ -0,0 +1,4 @@
+import { useContext } from "react";
+import { JobContext } from "../context/JobContext";
+
+export const useJob = () => useContext(JobContext);
```

Comment:
```suggestion
export const useJobData = () => useContext(JobContext);
```

Более правильно отображает суть